### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/01/day0.html?fbclid=IwAR3Xi-_awGGPpRZgA28CYl_Z9GSCx0sVpJr5geK2qDpG4qbpKpclC6bv8po

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#4] [Comments]
# https://mnfienen.github.io/talulat/2023/08/01/day0.html?fbclid=IwAR3Xi-_awGGPpRZgA28CYl_Z9GSCx0sVpJr5geK2qDpG4qbpKpclC6bv8po

It’s Day 0 - from the beach in SW Michigan to Sarnia, Ontario CANADA miles today: 0 total miles: 0 km today: 0 total km: 0

https://mnfienen.github.io/talulat/2023/08/01/day0.html?fbclid=IwAR3Xi-_awGGPpRZgA28CYl_Z9GSCx0sVpJr5geK2qDpG4qbpKpclC6bv8po

**Comment by Jlampe2** [comment]
What an exciting start and loved the timelapse! Will be following along and living vicariously through you two!
-Jake
